app/uk/gov/hmrc/agentsubscription/model/DesignatoryDetails.scala

```diff
@@ -24,7 +24,7 @@ case class DesignatoryDetails(person: Option[Person] = None)
 
 object DesignatoryDetails {
 
-  case class Person(dateOfBirth: Option[DateOfBirth] = None)
+  case class Person(lastName: Option[String] = None, dateOfBirth: Option[DateOfBirth] = None)
```

Comment:
why do you need to change this model as i don see its not used for companies house changes ?

Reply:
Yes it is - we need the last name to match at CH